ui/server/src/main/scala/pl/touk/nussknacker/ui/process/migrate/ProcessModelMigrator.scala

```diff
@@ -16,7 +16,8 @@ case class MigrationResult(process: CanonicalProcess, migrationsApplied: List[Pr
 
   def toUpdateAction(processId: ProcessId): UpdateProcessAction = UpdateProcessAction(processId,
     GraphProcess(ProcessMarshaller.toJson(process).noSpaces),
-    s"Migrations applied: ${migrationsApplied.map(_.description).mkString(", ")}"
+    s"Migrations applied: ${migrationsApplied.map(_.description).mkString(", ")}",
+    true
```

Comment:
Can u use named parameter here and in other cases of usage to make easier visible what this argument do without navigating to method declaration? BTW Intellij Idea highlighting such case as a code style issue

Reply:
You are right, I've added named parameters for readability.